### PR TITLE
Lint ignores

### PR DESCRIPTION
**Pull request by ahuth**
### Summary:

This PR tweaks how we ignore files for linting, so that `storybook-static` directories don't cause issues with linting. Running the axe tests (not merged in to master, yet) produces `storybook-static` dirs with a lot of contents that we don't want to lint.

### Test Plan:

CI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
     "build": "NODE_ENV=production lerna run build",
     "lint": "npm run lint:styles && npm run lint:scripts",
     "lint:fix": "npm run lint:styles:fix && npm run lint:scripts:fix",
-    "lint:styles": "stylelint packages",
+    "lint:styles": "stylelint --ignore-path .gitignore packages",
     "lint:styles:fix": "npm run lint:styles -- --fix",
-    "lint:scripts": "eslint --ext=js,jsx,ts,tsx .",
+    "lint:scripts": "eslint --ignore-path .gitignore --ext=js,jsx,ts,tsx .",
```
Comment (ahuth) [inline package.json:12]: I could've just added `storybook-static` to the .eslintignore and .stylelintignore files. It's kind of nice to just use the gitignore, though.

File: .node-version [hunk 2]
```diff
@@ -0,0 +1 @@
+12.16.3
```
Comment (ahuth) [inline .node-version:1]: Did not mean to commit this

File: package.json (hunk 1 above)
Comment (dcwither) [reply package.json:12]: yeah, that's awesome, full approve!

**Review by dcwither — APPROVED** [review]

**Comment by ahuth** [comment]
Thanks!

**Pull request merged**